src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -714,6 +723,12 @@ def _decode_asr(self, model_outputs, *, return_timestamps, return_language, time
             time_precision=time_precision,
         )
 
+    def create_initial_prompt_ids(self, text):
+        """Creates an initial prompt for generating text."""
+        intial_prompt_start_id = self.convert_tokens_to_ids("<|startofprev|>")
+        tokenized_text = self(" " + text.strip(), add_special_tokens=False)
```

Comment:
What is the reason for doing `" " + text.strip()`?

Reply:
This is what was done in the [Whisper implementation](https://github.com/openai/whisper/blob/b5851c6c40e753606765ac45b85b298e3ae9e00d/whisper/transcribe.py#L195). I believe the model formatting expects white spaces will be removed from the beginning and at the end of the string, with one space added at the start